<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpDownForm</class>
 <widget class="QDialog" name="UpDownForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>121</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Upload &amp; Download</string>
  </property>
  <property name="windowIcon">
   <iconset resource="interface.qrc">
    <normaloff>:/new/source/if_52_Cloud_Sync_183465.png</normaloff>:/new/source/if_52_Cloud_Sync_183465.png</iconset>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">Color : #FFFFFF;</string>
   </property>
   <property name="text">
    <string>Upload &amp; Download</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>241</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>0</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 2px;
Color : #FFFFFF;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNameLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>12</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileNameLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>65</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>18</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>File Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="actionButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>64</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Success</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="nameViewCheckBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Color : #FFFFFF;</string>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <zorder>closeButton</zorder>
  <zorder>titleLabel</zorder>
  <zorder>progressBar</zorder>
  <zorder>userNameLine</zorder>
  <zorder>fileNameLine</zorder>
  <zorder>actionButton</zorder>
  <zorder>nameViewCheckBox</zorder>
 </widget>
 <resources>
  <include location="interface.qrc"/>
 </resources>
 <connections/>
</ui>
